### DISCUSSION TITLE
配置重启模拟器

### DISCUSSION THREAD

**Discussion by Xiepohe** [LmeSzinc/AzurLaneAutoScript#2300] [Q&A]
看到alas里alas设置有重启模拟器
却没用在说明文档之中找到相关解释
求怎么填写参数及功能介绍
![image](https://user-images.githubusercontent.com/103975855/221577594-6b7b29c7-506b-46ea-95a3-5709410c2471.png)

**Comment by LmeSzinc** [comment]
#2068
